src/recipe/recipe.service.spec.ts

```diff
@@ -1,18 +1,188 @@
+          return Promise.resolve({
+            id: request.where.id,
+            createdAt: Date.now(),
+            title: faker.word.noun(),
+            description: faker.lorem.text(),
+            ingredients: faker.lorem.words(4),
+            preparation: faker.lorem.lines(5),
+          });
+        }),
+
+      findMany: jest.fn().mockImplementation(() => {
+        return Promise.resolve([
+          {
+            id: 0,
+            createdAt: Date.now(),
+            title: faker.word.noun(),
+            description: faker.lorem.text(),
+            ingredients: faker.lorem.words(4),
+            preparation: faker.lorem.lines(5),
+          },
+          {
+            id: 1,
+            createdAt: Date.now(),
+            title: faker.word.noun(),
+            description: faker.lorem.text(),
+            ingredients: faker.lorem.words(4),
+            preparation: faker.lorem.lines(5),
+          },
+        ]);
+      }),
+
+      update: jest
+        .fn()
+        .mockImplementation(
+          (request: { where: { id: number }; data: MockRecipeRequest }) => {
+            return Promise.resolve({
+              id: request.where.id,
+              createdAt: Date.now(),
+              ...request.data,
+            });
+          },
+        ),
+
+      delete: jest.fn().mockImplementation((_request) => {
+        return Promise.resolve();
+      }),
+    },
+  };
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [RecipeService],
+      providers: [
+        RecipeService,
+        {
+          provide: PrismaService,
+          useValue: mockPrismaService,
+        },
+      ],
     }).compile();
 
-    service = module.get<RecipeService>(RecipeService);
+    recipeService = module.get<RecipeService>(RecipeService);
   });
 
   it('should be defined', () => {
-    expect(service).toBeDefined();
+    expect(recipeService).toBeDefined();
+  });
+
+  describe('CreateRecipe', () => {
+    it('should create recipe', async () => {
+      //given
+      const request = {
+        title: faker.word.noun(),
+        description: faker.lorem.text(),
+        ingredients: faker.lorem.word(5),
+        preparation: faker.lorem.lines(3),
+      };
+
+      //when
+      const createdRecipe = await recipeService.createRecipe(request);
+
+      //then
+      expect(createdRecipe).toEqual(mockRecipe);
+    });
+  });
+
+  describe('FetchRecipe', () => {
+    it('should fetch recipe', async () => {
+      //given
+      const id = 777;
+
+      //when
+      const fetchedRecipe = await recipeService.fetchRecipe(id);
+
+      //then
+      expect(fetchedRecipe).toEqual({
+        id: expect.any(Number),
+        createdAt: expect.any(Number),
+        title: expect.any(String),
+        description: expect.any(String),
+        ingredients: expect.any(String),
+        preparation: expect.any(String),
+      });
```

Comment:
preferably use `expect.any` if you really don't know what the value of some field will be. But if you know, compare with that value (e.g. if you know a certain mock will be returned here and it's defined / accessible to this test file, import it and use it here)